release-manager/files/README.md

```diff
@@ -60,17 +60,31 @@ repositories:
 
 This type designates ODS components designed for _code development_. Such repositories are based on quickstarters whose names start with `be-`, `ds-`, or `fe-`, for _backend_, _data science_, and _frontend_, respectively. This is the default type.
 
+If you use this type ODS expects to find JUnit XML test results. If you do not have any test results the pipeline will fail. If you are deploying something where JUnit XML test results are not available consider using <<Repository Type: ods-service>>.
+
+#### Repository Type: ods-infra
+
+This type designates ODS components designed for _consuming on-prem or cloud services_ of arbitrary type using infrastructure as code. Such components are based on quickstarters whose names start with `infra-`.
+
+#### Repository Type: ods-saas-service
+
+This type designates ODS components designed for _documenting vendor-provided SaaS services_.
+
 #### Repository Type: ods-service
 
-This type designates ODS components designed for _running some service_. Examples include repositories based on the `be-gateway-nginx` quickstarter.
+This type designates ODS components designed for _running services_ of arbitrary type. Examples include repositories based on the `airflow-cluster` quickstarter.
```

Comment:
To me it's not clear why this is really different to the type ods as those components also run a.g. a webserver/webservice/API endpoint - or why is type ods different?

Which expectation is created by release manager to this component?

Reply:
The main difference I know of if the ods-service components do not need to have unit tests. Basically this type is for applications that do not have application code that needs to be tested and you just make use of it like ngnix